frontends/web/src/api/coins.ts

```diff
@@ -46,3 +46,41 @@ export type TAmount = {
 export const parseExternalBtcAmount = (amount: string): Promise<TAmount> => {
   return apiGet(`coins/btc/parse-external-amount?amount=${amount}`);
 };
+
+type TConvertCurrency = {
+  amount: string;
+  coinCode: CoinCode;
+  fiatUnit: Fiat;
+};
+
+type TConvertFromCurrencyResponse = {
+  success: true;
+  amount: string;
+} | {
+  success: false;
+  errMsg: string; // TODO: backend should return useful errorMessage
```

Comment:
backend currently may return 'internal error' or 'invalid amount', it should proabably return errorMessage and errorCode instead.

 https://github.com/digitalbitbox/bitbox-wallet-app/blob/43232f453dbe4a39a384f424515ecd9ae939b9ec/backend/handlers/handlers.go#L995-L998

Reply:
As we have other inconsistencies like this, I propose to keep the TODO comment and maybe one day take a look at all those error vs. errorMsg vs. errorMessage etc. consistent.

cc @Beerosagos ack?